### PR TITLE
[Wait for the new PR][Native] upload the example related with tensor_mux

### PR DESCRIPTION
**Pull request by helloahn**
this example can be activated after upgrading the new `sync_mode: refresh` of `tensor_mux`. The new mode will be applied soon at [nnstreamer](https://github.com/nnstreamer/nnstreamer) repo.

Signed-off-by: Hyoung Joo Ahn <hello.ahn@samsung.com>

**Self evaluation:**
1. Build test: [ ]Passed [ ]Failed [*]Skipped
2. Run test: [ ]Passed [ ]Failed [* ]Skipped

### CODE REVIEW

File: native/example_tensor_mux/example_tensor_mux.cc [hunk 1]
```diff
@@ -0,0 +1,344 @@
+/**
+ * @file	example_tensor_mux.c
+ * @date	22 Jun 2020
+ * @brief	Example with tensor_mux
+ * @author	HyoungJoo Ahn <hello.ahn@samsung.com>
+ * @bug		No known bugs.
+ * 
+ * appsrc ──> tee ──> tensor_filter:0 ──> tensor_mux ──> tensor_sink
```
Comment (leemgs) [inline native/example_tensor_mux/example_tensor_mux.cc:8]: Where is an expression of "**tensor_converter**" element that is used line 261?

File: native/meson.build [hunk 2]
```diff
@@ -8,6 +8,7 @@ endif
 if have_tensorflow_lite
 # TODO: Let's append the "example_repo_custom_easy" after sending SR to review.tizen.org
 # subdir('example_repo_custom_easy')
+#  subdir('example_tensor_mux')
```
Comment (leemgs) [inline native/meson.build:11]: double space? or single space? Looks good.

File: native/example_tensor_mux/example_tensor_mux.cc [hunk 3]
```diff
@@ -0,0 +1,344 @@
...
+    return -1;
+  }
+  TEST_MODE = argv[1][0];
+  switch (TEST_MODE){
+    case MODE_1:
+      sync_mode = g_strdup_printf ("slowest");
+      break;
+    
+    case MODE_2:
+      sync_mode = g_strdup_printf ("refresh");
```
Comment (leemgs) [inline native/example_tensor_mux/example_tensor_mux.cc:237]: If you cane, please describe a PR number and a definition to maintain a "refresh" mode.

File: native/example_tensor_mux/example_tensor_mux.cc [hunk 4]
```diff
@@ -0,0 +1,344 @@
...
+  g_assert (app);
+
+  /* init pipeline */
+  pipeline =
+      g_strdup_printf (
+        "tensor_mux name=mux sync_mode=%s ! tensor_sink name=sink async=false "
+
+        "appsrc name=appsrc caps=application/octet-stream,framerate=(fraction)0/1 ! "
+        "tensor_converter input-dim=1:1:1:1 input-type=uint8 ! tee name=t "
+          "t. ! queue ! tensor_filter framework=custom-easy model=model1 ! queue ! mux.sink_0 "
```
Comment (leemgs) [inline native/example_tensor_mux/example_tensor_mux.cc:262]: Why do we have to append "**queue**" in front of the " **! mux.sink_0**" statement?

File: native/example_tensor_mux/example_tensor_mux.cc (hunk 1 above)
Comment (helloahn) [reply native/example_tensor_mux/example_tensor_mux.cc:8]: it's missed. i'll updated it

File: native/example_tensor_mux/example_tensor_mux.cc (hunk 3 above)
Comment (helloahn) [reply native/example_tensor_mux/example_tensor_mux.cc:237]: I will

File: native/example_tensor_mux/example_tensor_mux.cc (hunk 4 above)
Comment (helloahn) [reply native/example_tensor_mux/example_tensor_mux.cc:262]: As I know, `queue` is recommended after `tee` and before `mux.sink_%d`. Honestly, I don't exactly know the reason yet. I'll find it.

File: native/meson.build (hunk 2 above)
Comment (helloahn) [reply native/meson.build]: it's fixed!

File: native/example_tensor_mux/example_tensor_mux.cc (hunk 1 above)
Comment (leemgs) [reply native/example_tensor_mux/example_tensor_mux.cc:8]: looks better.

**Pull request merged**